app/controllers/course/leaderboards_controller.rb

```diff
@@ -5,6 +5,7 @@ class Course::LeaderboardsController < Course::ComponentController
   before_action :add_leaderboard_breadcrumb
 
   def show #:nodoc:
+    load_course_users
```

Comment:
I would prefer having one hash or something, then the views extract out the variables from there, instead of having multiple instance variables, the instance variables make it a wee bit harder to test.